wire-ios/Wire-iOS Tests/TestPlans/AllTests.xctestplan

```diff
@@ -41,6 +41,7 @@
   "testTargets" : [
     {
       "skippedTests" : [
+        "SearchResultLabelTests",
```

Comment:
Is this flaky? If yes, did you create a ticket to look into it?

Reply:
yes, task IC-127